app/views/physical_server/_main.html.haml

```diff
@@ -0,0 +1,8 @@
+= render :partial => "layouts/flash_msg"
+.row
+  .col-md-12.col-lg-6
+    = render :partial => "shared/summary/textual", :locals => {:title => _("Properties"), :items => textual_group_properties}
```

Comment:
This is not how it is done any more. Please, see #274 that includes many exampled on how to define the groups.